src/views/migration/components/MigrationToolPhrase.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Trans } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import ExternalLink from '@kubevirt-utils/components/ExternalLink/ExternalLink';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { PackageManifestKind, SubscriptionsKind } from '../constant';
+
+type MigrationToolPhraseProps = {
+  operator: PackageManifestKind;
+  subscription: SubscriptionsKind;
+};
+
+const MigrationToolPhrase: React.FC<MigrationToolPhraseProps> = ({ operator, subscription }) => {
```

Comment:
we should only detect if it's installed and running, or if it's not,;
if it's installed we send users to the MTV application
if it's not installed completely we send users to the operator hub.

the differentiation between operator/subscription and instance depend on MTV and it's the responsibility of the MTV team to have documentation for that.
The best we can do is send user to the MTV docs.

Reply:
Can you define `not installed completely`?
We have 3 cases:
- not installed ( cluster has operator but not subscription ) -> Button is a link to the subscription page
- installed (operator and subscription ) -> Button is a link to the MTV page  something like `virt-openshift-mtv.apps.CLUSTER` 
-alert (no operator ) -> Alert

Reply:
we should identify if we have a path to a working MTV installation,
if no link available we assume MTV is not installed or not installed correctly (or completely, e.g. while installing)

if we have a link to local install (or in the future to an external install) we can have a link to the MTV applicatin
o/w we tell users we did not identify a working MTV installation and guide them to the documentation and to the operator hub in case they want to create a local install.